boskos/janitor/janitor.go

```diff
@@ -61,11 +68,11 @@ func setup(c boskosClient, janitorCount int, bufferSize int) chan string {
 	return buffer
 }
 
-func run(c boskosClient, buffer chan string) int {
+func run(c boskosClient, buffer chan string, res string) int {
 	totalAcquire := 0
 
 	for {
-		if proj, err := c.Acquire("project", "dirty", "cleaning"); err != nil {
+		if proj, err := c.Acquire(res, "dirty", "cleaning"); err != nil {
```

Comment:
IMO in my situation you'd simply change `--resource-type=gce-project,gke-project` to `--resource-type=gce-project,gke-project,large-project,aws-project` and `kubectl apply` that new flag value.

Your current design requires creating new deployments for the large and aws types